src/lib/ledger_catchup/normal_catchup.ml

```diff
@@ -588,10 +590,9 @@ let verify_transitions_and_build_breadcrumbs ~logger
           ]
         "build of breadcrumbs failed with $error" ;
       ( try
-          ignore
-            ( List.map transitions_with_initial_validation
-                ~f:Cached.invalidate_with_failure
-              : Mina_block.initial_valid_block Envelope.Incoming.t list )
+          List.iter transitions_with_initial_validation ~f:(fun (node, _vc) ->
+              (* TODO reject callback? *)
```

Comment:
See comment in super catchup related to handling of errors from `Breadcrumb_builder.build_subtrees_of_breadcrumbs`.